compiler/modules/bir/verify.bal

```diff
@@ -15,7 +15,7 @@ class VerifyContext {
         t:Context tc  = mod.getTypeContext();
         self.tc = tc;
         self.defn = defn;
-        self.anydataType = t:createAnydata(tc.env);
+        self.anydataType = t:createAnydata(tc);
```

Comment:
We can get rid of `anydataType` field now that we are memoizing anydata in the type context.

Reply:
fixed